test/falco_tests.yaml

```
@@ -86,6 +86,15 @@ trace_files: !mux
       - rules/rule_names_with_spaces.yaml
     trace_file: trace_files/cat_write.scap
 
+  rule_names_with_regex_chars:
+    detect: True
+    detect_level: WARNING
+    rules_file:
+      - rules/rule_names_with_regex_chars.yaml
+    detect_counts:
+      - 'Open From Cat (^$\.*+?()[]{}|)': 8
```

```suggestion
      - 'Open From Cat ($\.*+?()[]{}|^)': 8
```

What do you think of using the same order the chars are in the regex?

---

Sure, I reordered it originally when using a regex. Now that I'm using find_first_of() I'm using the same order for tests and code.